### PR TITLE
[SQL] `az sql server audit-policy`: Add sql server auditing support

### PR DESCRIPTION
**Pull request by omersavion**
**Description**  
Support sql server auditing in Azure CLI. Azure CLI currently supports sql database auditing (sql db audit-policy). This PR adds the support in server auditing too, using sql server audit-policy, which is similar to sql db audit-policy: [https://docs.microsoft.com/en-us/cli/azure/sql/db/audit-policy?view=azure-cli-latest](url)

In addition, updating the test of sql db audit-policy (test_sql_db_security_mgmt).

**Testing Guide**  
Testing can be done manually by using the commands "az sql server audit-policy show" or "az sql server audit-policy update" with the same arguments as sql db audit-policy, when --name is the name of the server, and there is no --server argument.
Testing can also be done by running the sql module test ("%ROOT%\src\azure-cli\azure\cli\command_modules\sql\tests\latest\test_sql_commands.py", specifically the test "test_sql_server_security_mgmt". 

**Release Note**
Added new commands `az sql server audit-policy show` and `az sql server audit-policy update` to view and manage audit policy for sql server.

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Auditing

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -130,6 +130,7 @@ Release History
 **SQL**
 
 * `az sql db ltr-policy/ltr-backup`: update/show long term retention policy, show/delete long term retention backups, restore long term retention backup (#12897)
+* Added new commands `az sql server audit-policy show` and `az sql server audit-policy update` to view and manage audit policy for sql server.
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst:133]: No need to change history.rst.  Please add release note in title or PR description box as guide.

Comment (omersavion) [reply src/azure-cli/HISTORY.rst:133]: Will be done on next commit.

File: src/azure-cli/azure/cli/command_modules/sql/commands.py [hunk 2]
```diff
@@ -251,6 +252,18 @@ def load_command_table(self, _):
         g.generic_update_command('update',
                                  custom_func_name='db_audit_policy_update')
 
+    server_blob_auditing_policies_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerBlobAuditingPoliciesOperations.{}',
+        client_factory=get_sql_server_blob_auditing_policies_operations)
+
+    with self.command_group('sql server audit-policy',
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/sql/commands.py:259]: could we mark these new command group as Preview for one sprint with `is_preview=True`?

**Comment by Juliehzl** [comment]
Hi @omersavion, could you also get @jaredmoo  approval?

File: src/azure-cli/azure/cli/command_modules/sql/commands.py (hunk 2 above)
Comment (omersavion) [reply src/azure-cli/azure/cli/command_modules/sql/commands.py:259]: No problem, I'll add it on next commit.

**Comment by Juliehzl** [comment]
@omersavion any update?

**Comment by omersavion** [comment]
Still waiting for @jaredmoo to approve this PR.

**Review by Juliehzl — APPROVED** [review]

**Comment by yonzhan** [comment]
@jaredmoo  there is only one day left for this PR to get merged. Please help review it ASAP.

**Comment by Juliehzl** [comment]
Because we will stop build today and there is no response from Jaredmoo, I will help merge the PR.

**Pull request merged**